geo/src/algorithm/relate/geomgraph/index/prepared_geometry.rs

```diff
@@ -5,7 +5,7 @@ use crate::GeometryCow;
 use crate::{GeoFloat, Relate};
 
 use std::cell::RefCell;
-use std::rc::Rc;
+use std::sync::Arc;
```

Comment:
All these Arc imports seem unused. Mr. Clipperton is so idiosyncratic with his hangups.